bigquery/nox.py

```diff
@@ -76,7 +76,7 @@ def system_tests(session, python_version):
         os.path.join('..', 'storage'),
         os.path.join('..', 'test_utils'),
     )
-    session.install('-e', '.')
```

Comment:
Looks like you undid the `session.install` change from the previous PR.

Reply:
Fixed.